lib/tasks/travis-ci.rake

```diff
@@ -0,0 +1,7 @@
+namespace :travis do
+  desc "Run all specs in except for the request specs"
+  RSpec::Core::RakeTask.new(:ci => :environment) do |task|
+    Rails.env = "test"
```

Comment:
Just FYI, I had to set `RAILS_ENV=test` explicitly (or change this line to `ENV['RAILS_ENV']="test"`) in order to get `rake travis:ci` to pass.